app/core/controllers/outgoing_emails.py

```diff
@@ -89,7 +89,12 @@ def post(self):
         email_contents = ForwardToAdminEmailHandler.write_email_contents(
```

Comment:
Nit: Should we change this to `ForwardToEmailHandler`?

Reply:
I thought about it, but I didn't want to give the impression that this forwards to arbitrary email addresses. Instead, I changed the docstring of the handler to be clearer about what sorts of email addresses the handler actually forwards to.